src/components/Extensibility/components-widget/Widget.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
```

Comment:
What's the reason for this file, when it's essentially identical to `components/Widget.js`?